compiler/src/cobalt/Parser/ExprParser.hs

```diff
@@ -310,7 +327,7 @@ newClassInstanceParser  = do
     try (rword "new")
     className <- typeRefParser
     arguments <- parens $ sepBy statementParser (symbol ",")
-    return $ (NewClassInstance className arguments)
+    return $ (NewClassInstance className (BlockStmt arguments))
```

Comment:
Since we have `BlockExpr` now, this should probably be parsed as expressions, not statements.

Reply:
I've updated this and `MethodCall` to use an `Expr` instead of `Stmt`.